tests/std/tests/concepts_matrix.lst

```diff
@@ -3,24 +3,24 @@
 
 RUNALL_INCLUDE .\prefix.lst
 RUNALL_CROSSLIST
-PM_CL="/w14640 /Zc:threadSafeInit- /EHsc /std:c++latest"
+PM_CL="/w14640 /Zc:threadSafeInit- /EHsc /std:c++latest /permissive-"
 RUNALL_CROSSLIST
 PM_CL="/MD /D_ITERATOR_DEBUG_LEVEL=0"
 PM_CL="/MD /D_ITERATOR_DEBUG_LEVEL=1"
 PM_CL="/MD /D_ITERATOR_DEBUG_LEVEL=0 /Zc:char8_t- /Zc:preprocessor"
-PM_CL="/MDd /D_ITERATOR_DEBUG_LEVEL=0 /permissive- /Zc:wchar_t-"
+PM_CL="/MDd /D_ITERATOR_DEBUG_LEVEL=0 /Zc:wchar_t-"
 PM_CL="/MDd /D_ITERATOR_DEBUG_LEVEL=1"
 PM_CL="/MDd /D_ITERATOR_DEBUG_LEVEL=2 /fp:except /Zc:preprocessor"
 PM_CL="/MT /D_ITERATOR_DEBUG_LEVEL=0 /await"
 PM_CL="/MT /D_ITERATOR_DEBUG_LEVEL=0 /analyze:only"
 PM_CL="/MT /D_ITERATOR_DEBUG_LEVEL=1"
 PM_CL="/MTd /D_ITERATOR_DEBUG_LEVEL=0 /fp:strict"
 PM_CL="/MTd /D_ITERATOR_DEBUG_LEVEL=1"
-PM_CL="/MTd /D_ITERATOR_DEBUG_LEVEL=2 /await"
+PM_CL="/MTd /D_ITERATOR_DEBUG_LEVEL=2 /await /permissive"

```

Comment:
This will emit `cl : Command line warning D9025 : overriding '/permissive-' with '/permissive'` unless they're going to stop emitting that. I think we should probably avoid crosslisting `/permissive-` here, although the alternative is verbose.